atlasdb-client/src/main/java/com/palantir/atlasdb/schema/stream/StreamStoreDefinition.java

```diff
@@ -32,29 +32,35 @@
     private final Map<String, TableDefinition> streamStoreTables;
     private final String shortName, longName;
     private final ValueType idType;
+    private final boolean compressStream;
 
     private int inMemoryThreshold;
 
-    StreamStoreDefinition(Map<String, TableDefinition> streamStoreTables, String shortName, String longName, ValueType idType, int inMemoryThreshold) {
+    StreamStoreDefinition(Map<String, TableDefinition> streamStoreTables, String shortName, String longName,
+            ValueType idType, int inMemoryThreshold, boolean compressStream) {
         this.streamStoreTables = streamStoreTables;
         this.shortName = shortName;
         this.longName = longName;
         this.idType = idType;
         this.inMemoryThreshold = inMemoryThreshold;
+        this.compressStream = compressStream;
     }
 
     public Map<String, TableDefinition> getTables() {
         return streamStoreTables;
     }
 
     public StreamStoreRenderer getRenderer(String packageName, String name) {
-        return new StreamStoreRenderer(Renderers.CamelCase(longName), idType, packageName, name, inMemoryThreshold);
+        return new StreamStoreRenderer(Renderers.CamelCase(longName), idType, packageName, name, inMemoryThreshold,
+                compressStream);
     }
 
-    public Multimap<String, Supplier<OnCleanupTask>> getCleanupTasks(String packageName, String name, StreamStoreRenderer renderer, Namespace namespace) {
+    public Multimap<String, Supplier<OnCleanupTask>> getCleanupTasks(String packageName, String name,
```

Comment:
Defer to you here - I didn't change anything on this method, was just cleaning up line lengths a little bit to get them <= 120. I'd prefer to have any further changes here be in another PR.